src/openzaak/components/catalogi/admin/mixins.py

```diff
@@ -77,14 +77,14 @@ def response_post_save_change(self, request, obj):
         else:
             return super().response_post_save_change(request, obj)
 
-    def is_published(self, obj):
+    def _is_published(self, obj):
```

Comment:
this function looks unnecessary. If you want it just to add description and boolean flag to the field, you can use `@admin.display` decorator for model method in the `ConceptMixin` model mixin

Reply:
It seems you cannot set the boolean flag in the model, it must be set in the admin.

Reply:
I think you can set it in the model for the method, but not for the property

Reply:
Can confirm it does work.